man/zip_source_function.mdoc

```diff
@@ -109,6 +109,12 @@ Must additionally support
 .Dv ZIP_SOURCE_TELL_WRITE ,
 and
 .Dv ZIP_SOURCE_REMOVE .
+.Pp
+On top of the above, supporting a pseudo-command
+.Dv ZIP_SOURCE_SUPPORTS_REOPEN
+will allow calling
+.Fn zip_fopen
+without the ZIP_FL_UNCHANGED flag to read the modified data.
```

Comment:
Sorry, I don't understand this. You could read the data multiple times without this PR - it was possible because window source takes care of repositioning the underlying source by seeking it to the desired offset before doing a read. The catch was that it was only possible for unmodified data, and since unmodified data is backed by a source that implements seeking, reading data multiple times always worked.